### PR TITLE
Exclude --force-rm from common bud cli flags

### PR DESCRIPTION
**Pull request by umohnani8**
Need this so we can override it to true by default in podman.

Used in fixing this https://github.com/containers/libpod/issues/1586

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @nalind PTAL

File: imagebuildah/build.go [hunk 1]
```diff
@@ -563,39 +563,39 @@ func NewExecutor(store storage.Store, options BuildOptions) (*Executor, error) {
...
-		defaultMountsFilePath:          options.DefaultMountsFilePath,
-		iidfile:                        options.IIDFile,
-		squash:                         options.Squash,
-		labels:                         append([]string{}, options.Labels...),
-		annotations:                    append([]string{}, options.Annotations...),
-		layers:                         options.Layers,
-		noCache:                        options.NoCache,
-		removeIntermediateCtrs:         options.RemoveIntermediateCtrs,
-		forceRmIntermediateCtrs:        options.ForceRmIntermediateCtrs,
+		quiet:                   options.Quiet,
```
Comment (rhatdan) [inline imagebuildah/build.go]: Why did these change?

File: pkg/cli/common.go [hunk 2]
```diff
@@ -74,6 +74,10 @@ var (
 			Name:  "layers",
 			Usage: fmt.Sprintf("cache intermediate layers during build. Use BUILDAH_LAYERS environment variable to override. (default %t)", UseLayers()),
 		},
+		cli.BoolFlag{
```
Comment (rhatdan) [inline pkg/cli/common.go]: Should this be in alphabetic order?

File: imagebuildah/build.go [hunk 3]
```diff
@@ -563,39 +563,39 @@ func NewExecutor(store storage.Store, options BuildOptions) (*Executor, error) {
...
-		commonBuildOptions:             options.CommonBuildOpts,
-		defaultMountsFilePath:          options.DefaultMountsFilePath,
-		iidfile:                        options.IIDFile,
-		squash:                         options.Squash,
-		labels:                         append([]string{}, options.Labels...),
-		annotations:                    append([]string{}, options.Annotations...),
-		layers:                         options.Layers,
-		noCache:                        options.NoCache,
-		removeIntermediateCtrs:         options.RemoveIntermediateCtrs,
-		forceRmIntermediateCtrs:        options.ForceRmIntermediateCtrs,
```
Comment (TomSweeneyRedHat) [inline imagebuildah/build.go]: I smell a gofmt done with go1.10 here.  Please be sure to run gofmt from go 1.11 or the CI will not be happy with you.

File: pkg/cli/common.go (hunk 2 above)
Comment (TomSweeneyRedHat) [reply pkg/cli/common.go]: yes please!

File: imagebuildah/build.go (hunk 1 above)
Comment (umohnani8) [reply imagebuildah/build.go]: yeah, I don't know why that happened. Fixing

File: pkg/cli/common.go (hunk 2 above)
Comment (umohnani8) [reply pkg/cli/common.go]: Fixed.

File: imagebuildah/build.go (hunk 3 above)
Comment (umohnani8) [reply imagebuildah/build.go]: Fixed.

File: imagebuildah/build.go (hunk 1 above)
Comment (umohnani8) [reply imagebuildah/build.go]: Fixed. vscode is being dumb for me, it keeps messing with gofmt :(

**Comment by umohnani8** [comment]
tests pass! @rhatdan can we get this in please.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**